space/space.go

```diff
@@ -83,16 +81,10 @@ func Createspace(db *sql.DB, space structs.Spacespec, berr binding.Errors, r ren
 		utils.ReportError(err, r)
 		return
 	}
```

Comment:
Rather than pull the image secret from vault copy the image secret from `akkeris-system` into the destination name space.